doc/releases/changelog-0.21.0.md

```diff
@@ -536,7 +536,7 @@
   [(#2111)](https://github.com/PennyLaneAI/pennylane/pull/2111)
 
 * An error is raised during QNode creation if backpropagation is requested on a device with
-  finite-shots specified.
+  finite shots specified.
```

Comment:
Would we like to switch to `finite-shots` in the entire code base?

Reply:
Same rule applies here. There is no correct "finite shots" vs "finite-shots", it all depends what way it is being used. In this case, the noun is "shots" and the modifier is "finite", so there's no role for a hyphen